tomviz/AlignWidget.h

```diff
@@ -77,34 +78,10 @@ protected slots:
 
   void sliceOffsetEdited(int slice, int offsetComponent);
 
-protected:
```

Comment:
I would rather we avoided moving everything to an internals class, we are not worried about inheritance/minimizing header includes. I would like to stick with normal members unless there is a good reason to switch.